Services/Dashboard/Block/classes/class.ilDashboardBlockGUI.php

```diff
@@ -186,7 +188,12 @@ protected function initAndShow(): void
             $this->http->sendResponse();
             $this->http->close();
         }
-        $this->returnToContext();
+        if ($this->parent === ilDashboardGUI::class) {
+            $this->returnToContext();
+            return '';
```

Comment:
Pls Remove that return. In case of a interruption in `returnToContext()` we dont want to see an empty page but preferably a falls back on the other view.